requirements.txt

```diff
@@ -7,3 +7,4 @@ django-cache-memoize~=0.1
 django-nested-admin~=3.3.2
 swapper~=1.1
 netaddr~=0.8
+python-dateutil>=2.7,<3.0
```

Comment:
It's possible that the library is already being pulled because it's a dependency of some other package, but I added it anyway with a relaxed version range to follow the **"explicit is better than implicit"** approach.
The version range is relaxed because it's a very stable library, at the same time we won't accept an eventual 3.0 automatically without manual testing.

Reply:
Understood the approach from the commit message and it seems much more readable and simple. I didn't use this library earlier, thanks for introducing an application for the same :)